charts/akeyless-k8s-secrets-injection/values.yaml

```diff
@@ -13,8 +13,11 @@ debug: false
 tlsCertsSecretName: vault-secrets-webhook-tls-certs
 
 image:
-  repository: akeyless/k8s-webhook-server
-  agentImage: akeyless/k8s-secrets-sidecar
+  repository: docker.registry-2.akeyless.io/k8s-webhook-server
+  agentImage: docker.registry-2.akeyless.io/k8s-secrets-sidecar
+#  repository: akeyless/k8s-webhook-server
```

Comment:
if it left as comment might be better mentioning that it can be a fallback

Reply:
Yes this is intentional, i'll add a comment